src/nativeAssetManager.js

```diff
@@ -378,7 +378,8 @@ export function newNativeAssetManager(win, pubUrl) {
     callback && callback();
     win.removeEventListener('message', replaceAssets);
     stopListening();
-    requestHeightResize(bid.adId, (document.body.clientHeight || document.body.offsetHeight), document.body.clientWidth);
+    const resize = () => requestHeightResize(bid.adId, (document.body.clientHeight || document.body.offsetHeight), document.body.clientWidth);
+    document.readyState === 'complete' ? resize() : window.onload = resize;
 
     if (typeof window.postRenderAd === 'function') {
```

Comment:
To me this looks correct:

at this point we are done rendering, and the `resize` function is scheduled to run when the DOM is done loading ( image sources and so on). If we run `postRenderAd` here it can manipulate the DOM and for example add another image that wold still come in time for `resize`.

Reply:
In one of our templates we retrieve the height of the ad, and we do that in postRenderAd. I was thinking that the height should be calculated when we're 100% sure that the resize has happened. So, if `document.readyState` is not `complete`, postRenderAd is executed before resize.